include/net/ieee802154_radio.h

```diff
@@ -26,14 +26,17 @@ extern "C" {
  * @{
```

Comment:
nit: Please update the commit title prefix: `net: ieee802154:`

Reply:
updated comment